test/__snapshots__/mathml-spec.js.snap

```diff
@@ -505,3 +499,55 @@ exports[`A MathML builder should use <munderover> for large operators 1`] = `
 </math>
 
 `;
+
+exports[`A MathML builder tags use <mlabeledtr> 1`] = `
+
+<math>
+  <semantics>
+    <mtable side="right">
```

Comment:
I assume that this causes the first `<mtd>` to be layed out to the right of the second.

Reply:
Yes. The order of the columns is strict: the first one should be the label, and the second one should be the formula, and then this controls the visual layout. See [mlabeledtr docs](https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mlabeledtr). (This is all new to me so I hope I got it all right!)

Reply:
I had a look at the docs.  Looks reasonable to me.